bzt/modules/blazemeter.py

```diff
@@ -1349,25 +1357,20 @@ def _merge_with_blazemeter_config(self):
 
     def prepare(self):
         self._merge_with_blazemeter_config()
-        if self.settings.get("dump-locations", False):
-            self.log.warning("Dumping available locations instead of running the test")
-            self._configure_client()
-            use_deprecated = self.settings.get("use-deprecated-api", True)
-            locations = self.user.available_locations(include_harbors=not use_deprecated)
-            for location_id in sorted(locations):
-                location = locations[location_id]
-                self.log.info("Location: %s\t%s", location_id, location['title'])
-            raise ManualShutdown("Done listing locations")
+        global TAURUS_TEST_TYPE  # FIXME: remove temporary solution
```

Comment:
This is temporary solution, we'll remove it in the future. I also don't like it.